### ISSUE TITLE
Review [license] section in modules

### ISSUE THREAD

**Issue by sbordet** [eclipse/jetty.project#7298]
**Jetty version(s)**
9.4+

**Description**
`[license]` sections of Jetty modules are sometimes missing (e.g. `test-keystore.mod`) sometimes only referencing a link to the license, sometimes the whole license text is pasted in the `[license]` section.

We need to decide on what we should do (link or whole text) and be consistent.

**Comment by joakime** [comment]
Traditionally all license components should be presented to the user.

The license text.
The license URL.
The `SPDX-License-Identifier`

**Comment by olamy** [comment]
maybe we could have a maven plugin checking if there is a `[license]` when there is one of the following sections `[files]` (when the section contains at least one line with `maven://`) or `[lib]`

**Comment by joakime** [comment]
Not a terrible idea.

**Comment by olamy** [comment]
> Traditionally all license components should be presented to the user.
> 
> The license text. The license URL. The `SPDX-License-Identifier`

@joakime do you mean every single mod file should have at least the Jetty license (dual eclipse/asf in the `[license]` section ?

**Comment by sbordet** [comment]
@olamy no, only the plugins that download non-Jetty libraries should have a `[license]` section.

We just need to verify that all the `*.mod` files that download non-Jetty libraries have the `[license]` section, and that it is correct.
